cves/kernel/CVE-2021-30002.yml

```diff
@@ -153,10 +153,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: '2020-12-19'
```

Comment:
Include the name of the developer who made the discovery here, considering you identified that below. Maybe if there is any interesting history in how/why it was discovered too